src/NzbDrone.Common.Test/WebClientTests.cs

```diff
@@ -23,7 +23,7 @@ public void DownloadString_should_be_able_to_dowload_text_file()
         [TestCase("http://")]
         public void DownloadString_should_throw_on_error(string url)
         {
-            Assert.Throws<ArgumentException>(() => Subject.DownloadString(url));
+            Assert.Catch(() => Subject.DownloadString(url));
```

Comment:
Does this fail? We want to assert the correct exception is thrown, not catch all exceptions.